articles/getting-started/setting-up-domains.md

```diff
@@ -16,12 +16,14 @@ The following are instructions for creating sending domains via the SparkPost UI
 
 1. Depending on what domain provider you are using, their DNS may look different, but how it works is essentially the same across all providers. There will be a TEXT field, and you can put the DKIM record in the text fields. We have provided you with many domain provider’s steps to go through to put your DKIM record into their records [here](https://www.sparkpost.com/docs/getting-started/getting-started-sparkpost/#prerequisites).
 
-1. Paste the "value" portion DKIM into your DNS TXT record for the hostname provided in the UI. Please note that the hostname for this record will be "domainkey._yoursendingdomain", and not the domain itself.
+1. To add the DKIM record to your DNS, first copy the "Hostname" portion provided in the _Set Up For Sending_ section of the SparkPost UI and paste it in the "Host" section of your DNS TXT record. This value should be `scphxxxx._domainkey.yourdomain.xxx`. Please note that some providers are particular about the "Host" and may require that the value be added as just `scphxxxx._domainkey`.
 
-1. Though the DKIM record (or domain key) looks long, do not break it into 2 or more lines. It may not look like DNS can't handle all of the characters, but it can.
+1. Next copy the "Value" portion provided in the _Set Up For Sending_ section of the SparkPost UI and paste it in the "Value" section of your DNS TXT record. Please note that even though the value (or domain key) looks long, do not break it into 2 or more lines or make any changes to the key. It may not look like DNS can handle all of the characters, but it can.
 
 1. It can take a bit of time for DKIM records to propagate. You can check the TTL (time to live) amount within your DNS provider – usually they are beside the record type. It can take up to that time both to be propagated and ready to go. Once they do, you may click the orange "Verify TXT Record" in the UI. If the domain validates successfully, this is what it should look like:
 
     ![](media/setting-up-domains/sending-domain-complete.png)
 
 1. You can have as many sending domains as you’d like. However, they can only be in ONE SparkPost account. Duplicates across multiple accounts are not permitted.
+
+1. Note that if you ever remove your domain from your SparkPost Account and re-add it, this will reset the TXT record for the domain in SparkPost, so you will need to update your DNS TXT record to verify the domain.
```

Comment:
Since this is a note, let's not add it to the numbered list. You could say `*Note:*` instead of `1. `.